### PR TITLE
CI/CD Workflow

### PR DESCRIPTION
**Pull request by luisan00**
# Add a CI/CD Workflow in several jobs

- Code-Spell: Search for misspelling words
- Build-Test: Check if the code compile

## Future work
- Unit-Tests
- Deploy documentation

### CODE REVIEW

**Comment by luisan00** [comment]
looks mr travis is not happy with codespell,it will need some improvements

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,36 @@
...
+before_install:
+  - sudo apt-get -y install doxygen
+
+install:
+  - pip install -U codespell
+  - pip install -U travis-sphinx
+  - pip install -U sphinx-rtd-theme
+  - pip install -U breathe
+  - pip install -U exhale
+
```
Comment (jeandudey) [inline .travis.yml]: This isn't used right now.

Comment (luisan00) [reply .travis.yml]: Not yet, but it will used in a near future

Comment (luisan00) [reply .travis.yml:19]: > Future work
Unit-Tests
Deploy documentation

**Comment by luisan00** [comment]
Closing, this pr is too old...

**Pull request closed without merging**